processing/src/main/java/io/druid/segment/ColumnSelectorFactory.java

```diff
@@ -30,4 +31,5 @@
   public FloatColumnSelector makeFloatColumnSelector(String columnName);
   public LongColumnSelector makeLongColumnSelector(String columnName);
   public ObjectColumnSelector makeObjectColumnSelector(String columnName);
+  public ValueType columnType(String columnName);
```

Comment:
Hmm, I think we can get away without having to make this interface change by rethinking how the variance aggregator shoudl work